issue #6 addinng skills section

aarushisingh2004 opened the pull request:
As per the the issue allotted to me, issue  #6 
 I had to make the 
->skills
->testimonials
->faq section 

Out of which, I have completed skills section,  for further sections ill send the pull request again ,please review my pr
The added section of skills look like:-
![skill1](https://github.com/SnowScriptWinterOfCode/PortfolioCreations/assets/120239056/43ef527b-90e7-466d-94e1-7d00d48fda38)
![skill2](https://github.com/SnowScriptWinterOfCode/PortfolioCreations/assets/120239056/f4051d49-60a6-483c-a7d4-82940801fef0)

aarushisingh2004 commented:
#6  issue assigned to me, skills section added please review  @i-am-SnehaChauhan

ayushichoudhary-19 commented on index.html:
```diff
@@ -14,6 +16,10 @@
                     <li><a href="#projects">Projects</a></li>
                     <li><a href="#contact">Contact</a></li>
+                    <!-- added the skills and FAQ section -->
+                    <li><a href="#Skills">Skills</a></li>
```
Let's maintain consistency. Could we keep the first letter of the ID name in lowercase, as in "#skills" and not "#Skills"? And the same goes for others too.

ayushichoudhary-19 commented on index.html:
```diff
@@ -66,8 +72,77 @@ <h3>Project 3</h3>
         </section>
 
+        <!-- Skills section -->
+        <section class="skills">
```
For the navbar link to work, we need to add `id="skills"` within this tag. Could you please incorporate that?

ayushichoudhary-19 reviewed (changes requested):
Please review the comments added to your committed code and incorporate the changes. Thank you!

aarushisingh2004 commented:
ok I will make the changes soon

aarushisingh2004 committed:
changed Skill to skill and id , all suggested changes done , please review

aarushisingh2004 commented:
please review @ayushichoudhary-19  I changed it to skill_section  and added skill_section to id , I kept the class name as "skill" only because I was using this class in css at many places , so to avoid mistake, please review , if any more changes are required

aarushisingh2004 committed:
added all changes suggested in previus pr

aarushisingh2004 committed:
merged all conflicts and added skills section

aarushisingh2004 commented:
please review @ayushichoudhary-19  , I have merged all conflicts and added changes suggested

ayushichoudhary-19 committed:
Merge branch 'main' into skills_section

ayushichoudhary-19 commented:
Great job, @aarushisingh2004! Your pull request looks good and I'll be merging it now!
